package.json

```diff
@@ -136,90 +139,33 @@
 					{
 						"type": "bashdb",
 						"request": "launch",
-						"name": "Bash-Debug (select script from list of sh files)",
-						"cwd": "${workspaceFolder}",
-						"program": "${command:SelectScriptName}",
-						"args": "",
-						"windows": {
-							"bashPath": "C:\\Windows\\sysnative\\bash.exe"
-						},
-						"linux": {
-							"bashPath": "bash"
-						},
+						"name": "Bash-Debug (simplest configuration)",
+						"program": "${file}",
```

Comment:
on windows bashpath should be "C:\\Windows\\sysnative\\bash.exe" by default

Reply:
Can I remove it? since its value is not unique -- `sysnative` or `System32`.

Reply:
Actually, in "initialConfigurations" i would keep the "system32" version, since this one seems to be correct when VSCode has same bitness as  OS. Please set the value as empty string only in line 97.